docs-src/0.5/en/cookbook/publishing.md

```diff
@@ -26,6 +26,17 @@ dx build --release
 - Add and commit with git
 - Push to GitHub
 
+## Web: Publishing with Netlify
+Set your Netlify build command to `cargo install dioxus-cli && dx build —release`.
```

Comment:
I think this should be `dx build —-release`

Reply:
I think his editor automatically converted it to the longer dash character `—`. 

Looks like GitHub did the same for your `--release`

Reply:
Yeahhhh totally just my editor... 😝 thanks for catching me